src/_playground/documentation/DocsTile/DocsTile.js

```diff
@@ -2,51 +2,62 @@ import { Button } from '../../../';
 import classnames from 'classnames';
 import { googlecode } from 'react-syntax-highlighter/styles/hljs';
 import PropTypes from 'prop-types';
+import React from 'react';
 import SyntaxHighlighter from 'react-syntax-highlighter';
 import { Toggle } from '../../../Toggle/Toggle';
-import React, { Component } from 'react';
 
-export class DocsTile extends Component {
+export class DocsTile extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            backgroundToggle: false
+            backgroundOn: true,
+            rightToLeft: false
         };
     }
 
-    handleToggle = () => {
-        this.setState({
-            backgroundToggle: !this.state.backgroundToggle
-        });
+    handleBackgroundToggle = () => {
+        this.setState(prevState => ({
+            backgroundOn: !prevState.backgroundOn
+        }));
+    };
+
+    handleRightToLeftToggle = () => {
+        this.setState(prevState => ({
+            rightToLeft: !prevState.rightToLeft
+        }));
     };
 
     render() {
         const { centered, children } = this.props;
+        const { backgroundOn, rightToLeft } = this.state;
 
-        const outerDivClasses = classnames(
-            'frDocs-Content__tile',
-            {
-                'frDocs-Content__tile-background': !this.state.backgroundToggle
-            }
-        );
+        const outerDivClasses = classnames('frDocs-Content__tile', {
+            'frDocs-Content__tile-background': backgroundOn
+        });
 
-        const innerDivClasses = classnames(
-            'fd-tile__content',
-            {
-                'frDocs-tile__centered': centered
-            }
-        );
+        const innerDivClasses = classnames('fd-tile__content', {
+            'frDocs-tile__centered': centered
+        });
 
         return (
             <div className={outerDivClasses}>
-                <Toggle
-                    className='frDocs-tile__background-toggle'
-                    inputProps={{ 'aria-label': 'Toggle background color' }}
-                    onChange={this.handleToggle}
-                    size='xs'>Toggle background</Toggle>
-                <div className={innerDivClasses}>
-                    {children}
+                <div className='frDocs-tile__features'>
+                    <Toggle
+                        className='frDocs-tile__feature'
+                        inputProps={{ 'aria-label': 'Toggle right to left' }}
+                        onChange={this.handleRightToLeftToggle}
+                        size='xs'>
+                        Toggle right to left
+                    </Toggle>
+                    <Toggle
+                        className='frDocs-tile__feature'
+                        inputProps={{ 'aria-label': 'Toggle background color' }}
+                        onChange={this.handleBackgroundToggle}
+                        size='xs'>
+                        Toggle background
+                    </Toggle>
```

Comment:
This is probably unfair to put this on you, but we should adjust the verbiage on both of these toggles.  A toggle is an on/off switch so the label should reflect that.  The label "Toggle ..." doesn't really state what the "on" or the "off" state will do.  In fact, the background one is actually probably backwards.  The "off" state shows a background and "on" does not.  I think the default state of both toggles should be "off".  That said, the default for "right to left" is LTR and the default for "background" is to have a background color.

How about these for the labels:
* Show RTL
* Hide background

That way, both can be "off" by default and then it's clear _what_ should happen when the toggle is "on".

**NOTE:** The `aria-label` values should be updated to match.

Reply:
I agree.  
I think the label `Show right to left` would be an improvement to `Show RTL` though